### ISSUE TITLE
only one instance of babel-polyfill is allowed

### ISSUE THREAD

**Issue by yogesum** [andrejewski/himalaya#33]
Throwing this error

```js
/home/yogesh/yogesum/api/node_modules/babel-polyfill/lib/index.js:10
  throw new Error("only one instance of babel-polyfill is allowed");
  ^

Error: only one instance of babel-polyfill is allowed
    at Object.<anonymous> (/home/yogesh/yogesum/api/node_modules/babel-polyfill/lib/index.js:10:9)
    at Module._compile (module.js:570:32)
    at Module._extensions..js (module.js:579:10)
    at Object.require.extensions.(anonymous function) [as .js] (/home/yogesh/yogesum/api/node_modules/babel-core/lib/api/register/node.js:214:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/yogesh/yogesum/api/node_modules/himalaya/lib/index.js:5:1)
    at Module._compile (module.js:570:32)
    at Module._extensions..js (module.js:579:10)
    at Object.require.extensions.(anonymous function) [as .js] (/home/yogesh/yogesum/api/node_modules/babel-core/lib/api/register/node.js:214:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/yogesh/yogesum/api/server/components/pdfgen/htmlParse.js:2:18)
    at Module._compile (module.js:570:32)
    at normalLoader (/home/yogesh/yogesum/api/node_modules/babel-core/lib/api/register/node.js:199:5)
    at Object.require.extensions.(anonymous function) [as .js] (/home/yogesh/yogesum/api/node_modules/babel-core/lib/api/register/node.js:216:7)
```

content of `himalaya/lib/index.js`:

```js
// line number 5
require('babel-polyfill');

var _lexer = require('./lexer');
```

himalaya version: `0.2.5` (project using babel for development)

**Comment by andrejewski** [comment]
Thanks for catching this. I was unaware but here is a discussion about fixing/improving the situation: https://github.com/babel/babel/issues/4019. An eventual fix there doesn't help us now. I've thought about this since including the polyfill in the first place. I will have a fix for this today.

**Comment by andrejewski** [comment]
This is fixed and published in `0.2.6`; thank you for the issue.
